wewatch-client/src/components/Lobby/Lobby.js

```diff
@@ -1,7 +1,7 @@
 import React, { useState, useEffect } from "react";
 import queryString from 'query-string';
 import io from "socket.io-client";
-import { Link } from "react-router-dom";
```

Comment:
Good catch removing the unused Link import. This keeps our code clean and understandable in the future!

Reply:
Thanks derek! great work helping me with that websocket bug! 🚀